packages/sample-apps/codegen/NativeMyModuleSpec.g.h

```diff
@@ -8,18 +8,12 @@
  */
 #pragma once
 
+#include "NativeMyModuleTypes.g.h"
```

Comment:
One use case I'm a little worried about is if in the future the API being called will want to add something to the struct that isn't currently available in the codegen.  If we put a #ifdef check around this include -- say `#ifndef NO_GENERATED_MYMODULE_TYPES`, then the codegen can also be used in a way that can just attach to existing structs, which would easily allow additional custom properties on the struct that are not available on the JS side.

Reply:
I can always skip this including